### DISCUSSION TITLE
r-placer discussions

### DISCUSSION THREAD

**Discussion by michaelshumshum** [michaelshumshum/r-placer#11] [Announcements]
discuss here

**Comment by omar2205** [comment]
Just verified a throwaway and the cool down is 5mins

**Reply by michaelshumshum** [reply]
looks good then, but it will add more work to account creation.

**Reply by omar2205** [reply]
yeah

**Reply by omar2205** [reply]
can you try with this https://www.1secmail.com/ ? it has an API to access the email

**Comment by omar2205** [comment]
So, 

1. Create an account & verify it
2. Add the account to the developers list
3. Get access_token
4. Send a setPixel query

**Comment by noseapus** [comment]
Hi, I got a bot working but i can only get 14 accounts to work at once. Maybe we can help each other out?

**Reply by michaelshumshum** [reply]
what is your problem? is it rate limiting?

**Reply by omar2205** [reply]
I would guess, you would need to use a proxy

**Reply by michaelshumshum** [reply]
using a proxy would add a ton of slowdown, and it doesn't seem to be a problem with ip. as some have pointed out in other repos, only some accounts are affected. there must be some defining factor that makes some bot accounts detected.

**Reply by michaelshumshum** [reply]
i guess the only solution would be to use accounts only a couple times. once it gets banned, replace it with another. i don't see a chance that they get unbanned.

it looks like reddit devs are on our tail.

**Comment by ekansh28** [comment]
hi, im getting an error
ModuleNotFoundError: No module named 'selenium'
im getting this error when im using python3

FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\sekan\\AppData\\Roaming\\gspread\\service_account.json'
im getting this error when im using just python account_maker.py

**Reply by michaelshumshum** [reply]
for the second error, you need to set up gspread first. follow the steps from [here](https://docs.gspread.org/en/latest/oauth2.html#for-bots-using-service-account).

im not sure what the problem is with the first one. i just updated the `requirements.txt`, so maybe try doing `pip install requirements.txt`.

**Reply by ekansh28** [reply]
i set up gspread but im getting this error 
`gspread.exceptions.APIError: {'errors': [{'domain': 'usageLimits', 'reason': 'accessNotConfigured', 'message': 'Access Not Configured. Drive API has not been used in project 628989745625 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/drive.googleapis.com/overview?project=628989745625 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.', 'extendedHelp': 'https://console.developers.google.com/apis/api/drive.googleapis.com/overview?project=628989745625'}], 'code': 403, 'message': 'Access Not Configured. Drive API has not been used in project 628989745625 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/drive.googleapis.com/overview?project=628989745625 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.'}`
i enabled the drive api too

**Reply by michaelshumshum** [reply]
well, the google sheets thing is not priority right now. please go find the solution to this somewhere else. i would help if i had the time, but the priority is at working on the bots themselves.

**Reply by ekansh28** [reply]
`{"errors":[{"message":"Ratelimited","path":["act"],"extensions":{"nextAvailablePixelTs":2147483647000}},{"message":"ratelimit` hit, next available: `2147483647000","path":["act"]}],"data":null}` what does this mean im getting this error after running bot.py

**Reply by michaelshumshum** [reply]
ur account got shadowbanned from r/place sadly.
